# HG changeset patch
# User hoodmane
# Date 1627635284 -7200
#      Fri Jul 30 10:54:44 2021 +0200
# Branch patch
# Node ID 8d1dd7efc05383a1ff7430ec3c64154a0ecd4be1
# Parent  e0e3f0fc068ab19d2a8f6daf1f89dea6a7912b8e
Change config settings

diff -r e0e3f0fc068a -r 8d1dd7efc053 c/_cffi_backend.c
--- a/c/_cffi_backend.c	Thu Jul 22 17:44:04 2021 +0200
+++ b/c/_cffi_backend.c	Fri Jul 30 10:54:44 2021 +0200
@@ -107,12 +107,12 @@
 
 #else
 
-# define CFFI_CHECK_FFI_CLOSURE_ALLOC 0
-# define CFFI_CHECK_FFI_CLOSURE_ALLOC_MAYBE 0
-# define CFFI_CHECK_FFI_PREP_CLOSURE_LOC 0
-# define CFFI_CHECK_FFI_PREP_CLOSURE_LOC_MAYBE 0
-# define CFFI_CHECK_FFI_PREP_CIF_VAR 0
-# define CFFI_CHECK_FFI_PREP_CIF_VAR_MAYBE 0
+# define CFFI_CHECK_FFI_CLOSURE_ALLOC 1
+# define CFFI_CHECK_FFI_CLOSURE_ALLOC_MAYBE 1
+# define CFFI_CHECK_FFI_PREP_CLOSURE_LOC 1
+# define CFFI_CHECK_FFI_PREP_CLOSURE_LOC_MAYBE 1
+# define CFFI_CHECK_FFI_PREP_CIF_VAR 1
+# define CFFI_CHECK_FFI_PREP_CIF_VAR_MAYBE 1
 
 #endif
 
